foreman-selinux-enable

```diff
@@ -17,20 +18,22 @@ is_redhat_6() {
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    /usr/sbin/semanage port -E > $TMP_PORTS
+
+    # Remove previously defined elasticsearch_port_t
+    # (this can be removed in future release)
+    grep elasticsearch_port_t $TMP_PORTS | sed s/-a/-d/g >> $TMP
+
     # Load policy
     /usr/sbin/semanage module -S $selinuxvariant \
       -a /usr/share/selinux/${selinuxvariant}/foreman.pp.bz2
 
     echo "boolean -m --on httpd_setrlimit" > $TMP
 
-    /usr/sbin/semanage port -E | grep -q elasticsearch_port_t || \
-      echo "port -a -t elasticsearch_port_t -p tcp 9200-9300" >> $TMP
-
-    /usr/sbin/semanage port -E | grep -q docker_port_t || \
-      echo "port -a -t docker_port_t -p tcp 2375-2376" >> $TMP
+    grep docker_port_t $TMP_PORTS | echo "port -a -t docker_port_t -p tcp 2375-2376" >> $TMP
```

Comment:
Should `|` be `||` here, so as to create the port only when `grep` returns a bad error code?  I don't think the output of grep needs piping.  Also prefer `grep -q`.